stdlib/public/core/Codable.swift

```diff
@@ -1833,6 +1833,14 @@ public struct CodingUserInfoKey : RawRepresentable, Equatable, Hashable {
         self.rawValue = rawValue
     }
 
+    /// Returns whether the given keys are equal.
+    ///
+    /// - parameter lhs: The key to compare against.
+    /// - parameter rhs: The key to compare with.
+    public static func ==(_ lhs: CodingUserInfoKey, _ rhs: CodingUserInfoKey) -> Bool {
```

Comment:
I'd meant to split this out into a different commit, but it made it in here anyway. `CodingUserInfoKey` was missing `==`, but the compiler didn't complain.